_posts/2016-05-20-tyxml4.md

```diff
@@ -0,0 +1,69 @@
+---
+title: TyXML 4.0.0
+layout: default
+author: The Ocsigen Team
+authorurl: https://ocsigen.org
+---
+
+It is with great pleasure that we are announcing the release of [TyXML 4.0.0][changelog]. The major features of this new release are a new syntax extension that allows to use the standard HTML syntax and an improved user experience for both old and new TyXML users.
```

Comment:
I would say "a new PPX syntax extension," – there already was `pa_tyxml`. Not sure how many people are aware of `pa_tyxml`, though.